sdk/lib/crt/wine/except_arm64.c

```diff
@@ -110,7 +110,6 @@ unsigned int CDECL __CxxQueryExceptionSize(void)
  *		_setjmp (MSVCRT.@)
  */
 __ASM_GLOBAL_FUNC(MSVCRT__setjmp,
-                  "mov x1, #0\n\t"  /* frame */
```

Comment:
@tkreuzer
> This matches what was done for x86_64 in
882980c17a9a033fa8e49a4c116af9583698d218.

Can you confirm that this is also true in our own version of the code?